hack/image/ansible/Dockerfile

```diff
@@ -16,7 +16,8 @@ ENV HOME=/opt/ansible \
 RUN yum clean all && rm -rf /var/cache/yum/* \
```

Comment:
Missing fragment ;-)

Reply:
I don't _think_ this needs a fragment since there are no user-facing changes. @asmacdo can correct me if I'm wrong.

Reply:
Hi @joelanford @asmacdo, 

`Using --ignore-installed with pip causes it to ignore the fact that setuptools has already been installed by rpm.`

I understand that a user was using the `setuptools` that was already installed locally. Now, the image will install it always and then it will use the newest version always. 

So, it will change the bahaviour for the users and needs a fragment. please, let me know if I am wrong?

Reply:
Users will not use setuptools. Setuptools is a python library for installing other python libraries. 

This is a bugfix-- we were already installing setuptools via rpm, but until this PR we were also (accidentally) installing setuptools with pip. With this change, pip will no longer install setuptools at all, and we will use the fixed version installed by the RPM.

Reply:
OK, but I still think that we should use a specific version instead of getting the latest one.

Reply:
Even though we dont say the version number, we are installing via rpm from the ubi repo-- we can count on that to be stable.